### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by zhiyaoma** [simontomaskarlsson/CycleGAN-Keras#1]
Hello, I found your experiment is very interesting, I want to run and learn it by myself. But I have some questions: Where is your dataset about train set and test set? Could you please send me one?

**Comment by simontomaskarlsson** [comment]
Hello zhiyaoma and thank you. I unfortunately can't give you the datasets used to generate the results you see in the ReadMe. What I recommend is however that you take a look at the original CycleGAN repository where you can find and download the datasets they have used. These are available at https://github.com/junyanz/CycleGAN#datasets

The 256x256 pixel images can be used with the Keras code directly. Note that other image sizes requires reshaping of the images.

**Comment by kingaza** [comment]
thx @simontomaskarlsson

**Comment by kingaza** [comment]
> Hello zhiyaoma and thank you. I unfortunately can't give you the datasets used to generate the results you see in the ReadMe. What I recommend is however that you take a look at the original CycleGAN repository where you can find and download the datasets they have used. These are available at https://github.com/junyanz/CycleGAN#datasets
> 
> The 256x256 pixel images can be used with the Keras code directly. Note that other image sizes requires reshaping of the images.

thx @simontomaskarlsson
